### DISCUSSION TITLE
[Component Request] Hamburger Menu

### DISCUSSION THREAD

**Discussion by SaadBazaz** [heroui-inc/heroui#1083] [Feature requests]
Something to go along with a nice side of fries.
Or with the Navbar, at least.

## Possible Implementations
A Hamburger icon
A side menu (#973 )

**Comment by jrgarciadev** [comment]
Hey the hamburger / toggle component is already included in the current version of NextUI https://github.com/nextui-org/nextui/blob/main/packages/react/src/navbar/index.ts#L14, we will keep it for the next version v2 https://github.com/nextui-org/nextui/discussions/1035

**Reply by SaadBazaz** [reply]
Oh dang I rebuilt one. Thnx.
